Tools/setup/requirements.txt

```diff
@@ -11,6 +11,7 @@ pandas>=0.21
 pkgconfig
 psutil
 pygments
+wheel
```

Comment:
It's already at the bottom, is the version wrong?

Reply:
typical of me, sorry I didn't notice it was already there, but it won't work if its towards the end, because of the way how pip -r works, it needs to be installed before pymavlink.

I'm force pushing a commit to fix this.